pkg/resource/deploy/deploytest/resourcemonitor.go

```diff
@@ -378,8 +381,9 @@ func (rm *ResourceMonitor) Invoke(tok tokens.ModuleMember, inputs resource.Prope
 
 	// unmarshal outputs
 	outs, err := plugin.UnmarshalProperties(resp.Return, plugin.MarshalOptions{
-		KeepUnknowns:  true,
-		KeepResources: true,
+		KeepUnknowns:     true,
+		KeepResources:    true,
+		KeepOutputValues: true,
```

Comment:
Why is it needed?

Reply:
Likewise same as the RegisterResource, could copy the comment to these three places?